src/systemtest/test_level2.py

```diff
@@ -1820,14 +1811,14 @@ def test_doesnt_return_infs(
         test_client,
         fake_data_with_inf,
     ):
-        (scanid,) = (fake_data_with_inf.scan_id,)
+        scanid = fake_data_with_inf.scan_id
         freqmode = fake_data_with_inf.freq_mode
-        url = f"/rest_api/v5/level2/development/{PROJECT_NAME}/{freqmode}/{scanid}/"  # noqa
+        url = f"/rest_api/v5/level2/development/{PROJECT_NAME}/{freqmode}/{scanid}/"
         r = test_client.get(
             url,
         )
         assert r.status_code == http.client.OK, r.text
-        assert "NaN" not in r.text
+        assert "NaN" not in r.text  ## TODO: nan is in text
```

Comment:
Document as an issue or fix?

Reply:
fixed already, but the comment what not removed.